constants/users.js

```diff
@@ -10,9 +10,14 @@ const USER_STATUS = {
   ACTIVE: "active",
 };
 
+const initialData = {
+  status: "ONBOARDING",
```

Comment:
as far as I am aware we have only 3 statuses. active ooo and idle

Reply:
https://discord.com/channels/673083527624916993/1069806199853285426/1098109013155123231
<img width="610" alt="image" src="https://user-images.githubusercontent.com/107163260/233432810-2dcda746-dc7a-4e9d-be14-714a5fee86e2.png">